lib/modules/@apostrophecms/schemas/src/apos/components/AposInputCheckboxes.stories.js

```diff
@@ -27,11 +27,11 @@ const field = {
   ]
 };
 
-const baseTemplate = '<AposInputCheckbox :field="field" :value="value" :status="status"/>';
+const baseTemplate = '<AposInputCheckboxes :field="field" :value="value" :status="status"/>';
 
 storiesOf('Inputs (Checkbox)', module)
   .add('Checkbox', () => ({
-    components: { AposInputCheckbox },
+    components: { AposInputCheckboxes },
```

Comment:
Can also be removed because of the automatic registration of Vue components for Apostrophe.